### PR TITLE
Update README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ufukyilmaz**
This PR updates the README and CONTRIBUTING files according to merge. In addition, I think that querying section should be added to the top of the place where Jet is explained. But I couldn't add this on my own. Also, I'm not sure about the style of content etc. feel free to request major changes in the content :) 

Checklist:
- [x] Labels and Milestone set

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,25 +25,41 @@ structures and concurrency primitives, including:
...
+Also, Hazelcast includes a production-ready
 [Raft](https://en.wikipedia.org/wiki/Raft_(computer_science))
 implementation which allows implementation of _linearizable_ constructs
 such as:
 
 * a distributed and reentrant lock implementation, called `FencedLock`
 * primitives for distributed computing such as `AtomicLong`,
 `AtomicReference` and `CountDownLatch`.
-
+
```
Comment (frant-hartm) [inline README.md:36]: Just whitespace.

**Review by frant-hartm — APPROVED** [review]
LGTM, but I would ask @JakeSCahill or @Serdaro to review.

File: README.md [hunk 2]
```diff
@@ -11,8 +11,10 @@
 ----
 
 Hazelcast is an open-source distributed in-memory data store and
```
Comment (JakeSCahill) [inline README.md:13]: The first sentence in this paragraph seems long and complex.

Perhaps we can break it down. Something like:

> Hazelcast is a distributed in-memory data store and computation platform.

> As an in-memory data store, Hazelcast gives you faster access to your data by storing it in memory. With more accessible data, you can leverage Hazelcast to process huge amounts of real-time events or static datasets with consistently low latency.

> To help you take advantage of all these features, Hazelcast comes with the following built-in data structures:

**Review by JakeSCahill — APPROVED** [review]
Generaly looks good to me 👍  Just one comment about the first sentence being quite long. This will be the first thing someone reads and it is complex to understand

**Pull request merged**